tracim/tracim/lib/email.py

```diff
@@ -156,11 +157,11 @@ def notify_created_account(
             )
         message = MIMEMultipart('alternative')
         message['Subject'] = subject
-        message['From'] = '{0} <{1}>'.format(
+        message['From'] = formataddr((
             self._global_config.EMAIL_NOTIFICATION_FROM_DEFAULT_LABEL,
-            self._global_config.EMAIL_NOTIFICATION_FROM_EMAIL,
+            self._global_config.EMAIL_NOTIFICATION_FROM_EMAIL)
         )
-        message['To'] = user.email
+        message['To'] = formataddr(('',user.email))
```

Comment:
you can show the real user name instead of showing email. this is better for user experience, don't you think? (ie `message['To'] = formataddr((user.get_display_name(),user.email)))`

Reply:
Done.